### PR TITLE
Update docs with sensor-collector connection health msg

### PR DESCRIPTION
**Pull request by ricekot**

### CODE REVIEW

File: docs/beta/api-observability/install-guide/install-sensor.md [hunk 1]
```diff
@@ -68,18 +68,18 @@ If the Sensor is healthy, you should see output similar to below.
 levoai-sensor-747fb4aaa9-gv8g9   1/1     Running   0             1m8s
 ```
 
-#### ii. Check connectivity
+#### ii. Verify connectivity with Satellite
```
Comment (hnataraj) [inline docs/beta/api-observability/install-guide/install-sensor.md:71]: @ricekot like 56 is already titled `Verify connectivity with Satellite`. Thats why the subheading I chose was `ii. Check connectivity` :-)

File: docs/beta/api-observability/install-guide/install-sensor.md [hunk 2]
```diff
@@ -68,18 +68,18 @@ If the Sensor is healthy, you should see output similar to below.
...
-kubectl -n levoai logs <levoai-sensor pod name> | grep "TBD"
+kubectl -n levoai logs <levoai-sensor pod name> | grep "Initial connection with Collector"
 ```
-If connectivity is healthy, you will see output similar to below.
+If connectivity is healthy, you should see the following output:
 
-```bash
-TBD
+```
+Initial connection with Collector was successful.
```
Comment (hnataraj) [inline docs/beta/api-observability/install-guide/install-sensor.md:82]: @ricekot Is this the entire log line, or just a snippet? I would put the entire log line, and say output similar to below.

File: docs/beta/api-observability/install-guide/install-sensor.md [hunk 3]
```diff
@@ -111,7 +111,17 @@ sudo docker run --restart unless-stopped \
...
+Execute the following command to check for connectivity health:
+
+```bash
+# Please specify the actual container name for levoai-sensor below
+docker logs <levoai-sensor container name> | grep "Initial connection with Collector"
+```
+If connectivity is healthy, you should see the following output:
+
+```bash
+Initial connection with Collector was successful.
```
Comment (hnataraj) [inline docs/beta/api-observability/install-guide/install-sensor.md:123]: @ricekot Is this the entire log line, or just a snippet? I would put the entire log line, and say output similar to below.

File: docs/beta/api-observability/install-guide/install-sensor.md [hunk 4]
```diff
@@ -215,11 +225,19 @@ sudo cat /var/log/syslog | grep 'levo-ebpf-sensor'
 ```
 
 #### Connection Success
-TBD
+If connectivity is healthy, you should see the following output:
+
+```bash
+Initial connection with Collector was successful.
```
Comment (hnataraj) [inline docs/beta/api-observability/install-guide/install-sensor.md:231]: Is this the entire log line, or just a snippet? I would put the entire log line, and say output similar to below.

File: docs/beta/api-observability/quickstart.md [hunk 5]
```diff
@@ -168,18 +168,18 @@ If the Sensor is healthy, you should see output similar to below.
 ```bash
 levoai-sensor-747fb4aaa9-gv8g9   1/1     Running   0             1m8s
 ```
-#### ii. Check connectivity
+#### ii. Check connectivity with Satellite
```
Comment (hnataraj) [inline docs/beta/api-observability/quickstart.md:171]: See comments above. Same applies here.

File: docs/beta/api-observability/install-guide/install-sensor.md (hunk 1 above)
Comment (ricekot) [reply docs/beta/api-observability/install-guide/install-sensor.md:71]: Ah, my bad. I'll update it.

**Comment by ricekot** [comment]
Addressed comments.

**Pull request merged**